ironfish-cli/src/commands/chain/download.ts

```diff
@@ -245,10 +243,43 @@ export default class Download extends IronfishCommand {
   }
 
   async unzip(source: string, dest: string): Promise<void> {
+    let totalEntries = 0
+    let extracted = 0
+
+    const progressBar = CliUx.ux.progress({
+      barCompleteChar: '\u2588',
+      barIncompleteChar: '\u2591',
+      format:
+        'Unzipping snapshot: [{bar}] {percentage}% | {value} / {total} entries | {speed}/s | ETA: {estimate}',
+    }) as ProgressBar
+
+    const speed = new Meter()
+
+    progressBar.start(totalEntries, 0, {
+      speed: '0',
+      estimate: TimeUtils.renderEstimate(0, 0, 0),
+    })
+    speed.start()
+
+    tar.list({
+      file: source,
+      onentry: (_) => progressBar.setTotal(++totalEntries),
+    })
```

Comment:
Is this fast or should we add a CLI status action around this?

Reply:
It's not that fast -- It takes a couple minutes for the total in the progress bar to stop updating. It slows the unzip down a bit during that time

What would adding a CLI status action around it look like?